### DISCUSSION TITLE
Is there a way to map cameras and sort of join login?

### DISCUSSION THREAD

**Discussion by dopeytree** [blakeblackshear/frigate#7960] [General]
Is there a way to map cameras and sort of join logic?

Noticed some of my cameras pick up the cat but then they seem to get missed on the camera right next door.

For a test I test up a camera on 2 sides of a fence.

One is at ground level and the other wise is at headheight.

I wondered if there is a way to have joined cameras say hey I've just detected an object 'cat' be on the look out for this object.

**Comment by NickM-27** [comment]
seems like you're referring to what is discussed at https://github.com/blakeblackshear/frigate/discussions/7394

**Reply by dopeytree** [reply]
ah yes.

Though how do you send missing data to frigate+ ?

Would I need to record all motion and filter through it?

**Reply by NickM-27** [reply]
that was discussed here https://github.com/blakeblackshear/frigate/discussions/7932#discussioncomment-7097047
